modular_skyrat/modules/assault_operatives/code/armaments/armament_explosives.dm

```diff
@@ -1,22 +1,30 @@
-// EXPLOSIVES
-#define ARMAMENT_CATEGORY_EXPLOSIVES "Explosives"
-#define ARMAMENT_CATEGORY_EXPLOSIVESLIMIT 4
-
 /datum/armament_entry/assault_operatives/explosives
-	category = ARMAMENT_CATEGORY_EXPLOSIVES
-	category_item_limit = ARMAMENT_CATEGORY_EXPLOSIVESLIMIT
-	max_purchase = 8
+	category = "Explosive Devices"
+	category_item_limit = 4
+	max_purchase = 4
 
 /datum/armament_entry/assault_operatives/explosives/minibomb
 	item_type = /obj/item/grenade/syndieminibomb
 	cost = 3
 
-/datum/armament_entry/assault_operatives/explosives/frag
-	item_type = /obj/item/grenade/frag
-	cost = 2
+/datum/armament_entry/assault_operatives/explosives/bioterror
+	item_type = /obj/item/grenade/chem_grenade/bioterrorfoam
```

Comment:
bioterror ain't worth 3

Reply:
ive never used bioterror ever its only 3 points because it sounds scary